CHANGELOG.md

```diff
@@ -1,27 +1,35 @@
 # nf-core/scrnaseq: Changelog
 
-## dev / unreleased
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-### Fixes
+## [Unpublished Version / DEV]
+
+### `Added`
+
+* FastQC to QC raw reads
+
+### `Fixed`
 
 * [25](https://github.com/nf-core/scrnaseq/issues/25) Fix small documentation error with wrong parameter for txp2gene
 
-### Dependency Updates
+### `Dependencies`
 
-* Salmon to 1.1.0
-* Samtools to 1.10
-* GFFRead to 0.11.7
-* Kallisto to 0.46.2
-* BUSTools to 0.40.0
+* Added FastQC `0.11.9`
+* Update Salmon `1.0.0` -> `1.1.0`
+* Update Samtools `1.9` -> `1.10`
+* Update GFFRead `0.11.6` -> `0.11.7`
+* Update Kallisto `0.46.0` -> `0.46.2`
```

Comment:
@winni2k commented these days in one of the issues, that there is a problem with the latest kallisto version - maybe we stick to the 0.46.1 instead? https://github.com/nf-core/scrnaseq/issues/21#issuecomment-586206252

Reply:
Oh, sorry, it wasn't me who bumped kallisto. That change is already implemented in dev. I just re-formatted the line in the `CHANGELOG.md` file. What do you want me to do, then? Should I fix it and change it back to 0.46.0? Or maybe to 0.46.1 like winni2k suggested?

Reply:
I think we should go back to 0.46.1 then - if you can do that in your PR, that would be fab! :)

Reply:
Done. I've already pushed the changes. I guess they should be reflected here eventually?

Reply:
They automatically show up here immediately after your push :-)